examples/HighchartsGanttWithHighstockNavigation/App.js

```diff
@@ -0,0 +1,88 @@
+import React, {Component} from 'react';
+import Highcharts from 'highcharts/highcharts-gantt';
+import {
+  GanttSeries,
+  HighchartsGanttChart,
+  Subtitle,
+  Title,
+  withHighcharts,
+  YAxis
+} from 'react-jsx-highcharts-gantt';
+import ExampleCode from '../utils/ExampleCode';
+import code from './exampleCode';
+import {Scrollbar, RangeSelector, Navigator} from 'react-jsx-highstock';
+
+
+class App extends Component {
+
+  render() {
+
+
+    return (
+      <div className="app">
+        <HighchartsGanttChart>
+          <Title>Gantt Chart with Highstock Navigation</Title>
+          <YAxis
+            uniqueNames={true}
```

Comment:
True is the default argument for a React prop, so this can just be `<YAxis uniqueNames>`

Reply:
OK, I've removed "={true}" here and other similar places, so that the code is easier to read.